build/generateImplicitAPI.ts

```diff
@@ -216,22 +216,29 @@ function createImplicitMethodsGo(){
         const exchange = exchanges[index];
         const methodNames = storedCamelCaseMethods[exchange];
 
+        const reusableMethod = [
+            `func (this *${exchange}) callEndpointAsync(endpointName string, args ...interface{}) <-chan interface{} {`,
```

Comment:
Should we add `//go:noinline` here? I wonder the compiler would inline the function code.

Reply:
@ttodua but really does it ? we have to check that first

Reply:
> @ttodua but really does it ? we have to check that first

here what i mean: https://discord.com/channels/690203284119617602/1376639875725660241/1379402939168456746